addons/axe/functions/fn_cutDownTree.sqf

```diff
@@ -40,7 +40,7 @@ private _progressCheck = {
     _args params ["_treeObject", "_lastSoundEffectTime", "_unit"];
 
     if (_passedTime > (_lastSoundEffectTime + SOUND_CLIP_TIME_SPACEING)) then {
-        playSound3D ["x\grad\addons\axe\sound\chop.ogg", objNull, false, (getPosASL _unit), 1, 1, 80];
+        playSound3D [format ["x\grad\addons\axe\sound\chop_%1.ogg", (round(random 25)) +1], objNull, false, (getPosASL _unit), 1, 1, 80];
```

Comment:
❤️ 

aber: führt dazu daß "26" und "1" jeweils nur halb so oft gewählt werden wie die anderen Werte.

`round random 2` wird 
* im Intervall `]0; 0.5[` zu 0 , 
* im Intervall `]0.5; 1.5[` zu 1 
* im Intervall `]1.5; 2.0[` zu 2

bei gleichmäßiger Verteilung der Zufallswerte halb so oft 0 oder 2 wie 1

----

Stattdessen nimm `(floor random 25) + 1`

Reply:
Und was mach ich wenn random 26 aus gibt?
Das ist dann 26 +1 = 27.

Reply:
random gibt nicht 26 aus! random gibt maximal 25.9999 aus.

siehe auch den Kommentar vonm *Lou Montana* auf https://community.bistudio.com/wiki/random

Reply:
Jemand anderes der meinung Fussels methode ist besser?

Reply:
ja, aber den unterschied hören wird niemand :D

Reply:
Hören wirds keiner bei der Anzahl an Sounds, das is wohl richtig. Darum hab ich auch den Haken des Einverständnisses unter den PR gesetzt.

Aber es geht hier nicht um Meinung sondern um das Verständnis wie `random` funktioniert. 🤓